### ISSUE TITLE
AppImages can't be downloaded/installed using the NX Software Center

### ISSUE THREAD

**Issue by zhainhart** [Nitrux/nx-software-center#138]
**Describe the bug**
Download/install software from nx-software-center seems not working, click the package and do nothing, although the path is correct (taken from the terminal). 

**To Reproduce**
1. Search for Nextcloud
2. Click the app name from the search list
3. After the page displays the package information, click the "get" button.
4. The page will scroll to the list of app versions, 
5. Click one of them and the system does nothing

**Expected behavior**
The software installed

**Terminal Log**

```
APPS RESPONSES :: "Nextcloud - AppImage"
Setting app model "Nextcloud - AppImage"
(QVariant(QVariantMap, QMap(("pic", QVariant(QString, "https://images.pling.com/cache/770x540-0/img/00/00/70/99/16/2144213/logo-nextcloud-blue.png"))("smallpic", QVariant(QString, "https://images.pling.com/cache/770x540-0/img/00/00/70/99/16/2144213/logo-nextcloud-blue.png")))))
(QVariant(QVariantMap, QMap(("gpgfingerprint", QVariant(QString, ""))("gpgsignature", QVariant(QString, ""))("link", QVariant(QString, "https://files04.pling.com/api/files/download/j/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6IjE3MTIyNzg4MjUiLCJvIjoiMSIsInMiOiJhYzViNDZjMzdiZTI4MTdiMTI2M2U4NDJmNGY0NTQzMDBjZGFkMjljYjExMTA4OWVjMWIxMmUyMDk1YWY1ZjIxZmU1YzY4OWQ1NDFmMzhiZWE4ZWIyN2VlNDNkMGVlZWNjM2RjMjYzYjQzNTUwODhkMjBkZTYyNjNlMDUwODU0MyIsInQiOjE3MTQ1NTc4NzksInN0ZnAiOm51bGwsInN0aXAiOiIyMDIuODAuMjE5LjcwIn0.MQcKIIsxA4RvQw-gK9BhOqYdt8BujfHL0ScmU_tNhkI/Nextcloud-3.12.3-x86-64.AppImage"))("name", QVariant(QString, "Nextcloud-3.12.3-x86-64.AppImage"))("packagearch", QVariant(QString, ""))("packagename", QVariant(QString, ""))("packagetype", QVariant(QString, ""))("price", QVariant(QString, "0"))("repository", QVariant(QString, ""))("size", QVariant(QString, "149025"))("tags", QVariant(QString, "data##mimetype=application/x-executable"))("type", QVariant(QString, ""))("way", QVariant(QString, "")))))
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/GalleryRollTemplate.qml:114:17: QML Image: Connection closed
qrc:/maui/kit/GalleryRollTemplate.qml:114:17: QML Image: Connection closed
qrc:/maui/kit/GalleryRollTemplate.qml:114:17: QML Image: Connection closed
qrc:/maui/kit/GalleryRollTemplate.qml:114:17: QML Image: Connection closed
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/ListItemTemplate.qml: Object or context destroyed during incubation
qrc:/maui/kit/GalleryRollTemplate.qml:114:17: QML Image: Connection closed
```

**Desktop (please complete the following information):**
 - OS: Nitrux 3.4.1
 - NX Software Center Version 1.1.1

**Comment by UriHerrera** [comment]
![Screenshot_20240501_143938](https://github.com/Nitrux/nx-software-center/assets/3053525/29235cd3-24fc-4d8d-b6bc-4fff3b78095b)

I can confirm the problem. The Software Center lists the content correctly; search works; however, clicking the button to get the AppImage does nothing.

In the screenshot, the last lines are from the moment I clicked the button.

**Comment by probal31** [comment]
Changes/Fix made to support mauikit version 3.1.0

@UriHerrera Could you please check once? Thanks..

**Comment by UriHerrera** [comment]
Fixed in https://github.com/Nitrux/nx-software-center/commit/1c755326b003c91ac9c10d79bc56912fad7dbad0

The latest [NX SC AppImage](https://github.com/Nitrux/nx-software-center/releases) allows the search and downloading of AppImages from AppImageHub.com
